documentation/assemblies/upgrading/assembly-upgrade-kafka-versions.adoc

```diff
@@ -19,7 +19,8 @@ The Cluster Operator initiates rolling updates based on the Kafka cluster config
 ¦The Cluster Operator initiates...
 
 ¦Both the `inter.broker.protocol.version` and the `log.message.format.version`.
-¦A single rolling update.
+¦A rolling update for the update of each property, which must be updated separately.
+The `inter.broker.protocol.version` must be updated first, followed by the `log.message.format.version`
```

Comment:
I'm not sure this is clear. When both `inter.broker.protocol.version` and `log.message.format.version` are set in the Kafka CR, there will be a single rolling update done automatically. And then the users should manually update the `inter.broker.protocol.version` value, wait for another rolling update and update `log.message.format.version`. But I do not think I would get that from this table.

Reply:
I've changed to:

¦Both the `inter.broker.protocol.version` and the `log.message.format.version`.
¦A single rolling update. After the update, the `inter.broker.protocol.version` must be updated manually, followed by `log.message.format.version`, 
which triggers two more rolling updates.